source/manual/upgrade-to-sentry.html.md

```diff
@@ -17,9 +17,9 @@ To upgrade make a PR that does the following:
 1. Remove the `airbrake` gem from the app
 1. Remove the Airbrake initialiser `rm config/initializers/airbrake.rb`
 1. Add `gem "govuk_app_config", "~> 0.2.0"` to your Gemfile
-1. Use at least `gem "govuk_sidekiq", "~> 2.0.0"` to maintain error reporting
-1. Use at least `gem "slimmer", "~> 11.0.2"`, otherwise [errors won't be reported][slimmer]
-1. Use at least `gem "gds-api-adapters", "~> 47.9.1"` to improve error reporting
+1. If you use `govuk_sidekiq`, upgrade at least `gem "govuk_sidekiq", "~> 2.0.0"` to maintain error reporting
+1. If you use `slimmer`, upgrade to at least `gem "slimmer", "~> 11.0.2"`, otherwise [errors won't be reported][slimmer]
```

Comment:
Is there a reason this differs from the line above? ('to maintain error reporting' vs 'otherwise errors won't be reported')